nimbus_nlp/NIMBUS_NLP.py

```diff
@@ -1,16 +1,3 @@
-
-import nltk
-import numpy as np
-import os
-import pandas as pd
-import re
-import sklearn.neighbors
-import spacy
-import sys
-
-from google.api_core.client_options import ClientOptions
-from google.cloud import automl_v1
-from google.cloud.automl_v1.proto import service_pb2
```

Comment:
**{{QUESTION}}**

amazing, all of these were unused imports!? This surely should have reduced our memory usage, right? 

Why do you think the overall memory usage in the graph did not decrease? @cameron-toy

Reply:
I think the slight difference between peak memory in the two graphs can be explained by the removal of the google cloud imports. All the others are used in question_classifier, and Python gracefully handles redundant imports so not much is saved there.

Reply:
Interesting! [Yes, python does gracefully handle double imports](https://stackoverflow.com/a/2029546)